spec/factories/events.rb

```diff
@@ -1,8 +1,14 @@
 FactoryBot.define do
   factory :event do
-    date_time { DateTime.now }
+    date_time { DateTime.now + rand(10).days }
     activity { Faker::Hobby.activity }
     user
     gym
   end
 end
+
+def event_with_invitation
```

Comment:
Can an event be created with an invitation?

Reply:
I think this is something we should flag for later.  Perhaps we include a validation on either the FE or BE.